apps/website/src/pages/docs/expandableblock.mdx

```diff
@@ -9,67 +9,83 @@ group: core
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-expandableblock' />
-
 <LiveExample src='ExpandableBlock.main.tsx'>
   <AllExamples.ExpandableBlockMainExample client:load />
 </LiveExample>
 
-Expandable blocks, called expandable rows when inside a table, are used to progressively disclose information. Read this article to learn more about progressive disclosure. Expandable blocks and rows should not be used with flattened hierarchy or information panels. This type of navigation allows the user to see content inside a parent row or block.
+The expandable block has a label reflecting its content. On click, it expands to reveal more content.
+
+## Variants
 
-## Appearance
+### With caption
+
+<LiveExample src='ExpandableBlock.withcaption.tsx'>
+  <AllExamples.ExpandableBlockWithCaptionExample client:load />
+</LiveExample>
 
-### Expandable rows
+Users can optionally display descriptor text underneath the title.
+
+### Accordion
+
+<LiveExample src='ExpandableBlock.accordion.tsx'>
+  <AllExamples.ExpandableBlockAccordionExample client:load />
+</LiveExample>
 
-Expandable rows resemble normal table rows that can be expanded to reveal nested data. An expandable row could optionally be selectable. Because expandable rows are part of a table, the rows must sort based on the table column sort applied. This means expandable rows can not re-arranged, the user must interact with the table header to re-arrange.
+Expandable blocks can also stack on top of each other in an accordion to make large sets of information easier to read
 
-Expandable rows should not be confused with a tree hierarchy. Unlike trees, they follow a single branch pattern. If you have trouble telling the two apart, the Expandable block VS Hierarchy tree section explains what makes them different.
+### With status
 
-There can be several child elements nested in an expandable row. These indented elements all coexist on the same level without further indentation. If you require more levels, a tree hierarchy is a better pattern for this interaction.
+<LiveExample src='ExpandableBlock.status.tsx'>
+  <AllExamples.ExpandableBlockStatusExample client:load />
+</LiveExample>
 
-The child elements don’t reflect the column titles. They are expected to retain the same properties as the parent row (in the example below, the owner). It’s unnecessary to repeat the properties for the child items because of this, so do not include them in the expanded row area.
+An expandable block could display status, an icon, or other helpful information at a glance right aligned within the block header.
```

Comment:
The old documentation site said the following:

`An expandable block could display status, a total count, or other helpful information at a glance right aligned within the block header`

I tried to find a way to display the count, but the count seems off-center so I removed it as an example

![ExpandableBlockTotalCountBug](https://user-images.githubusercontent.com/78564221/229947570-59af6e6b-5a04-4439-b08f-c9271917979b.gif)